components/component.go

```diff
@@ -195,3 +200,34 @@ func (c *Component) UpdatePrometheusConfig(_ client.Client, logger logr.Logger,
 
 	return err
 }
+
+func (c *Component) GetReleaseVersion(defaultManifestPath string, componentName string) status.ComponentStatus {
```

Comment:
Frankly, I do not see a point to have it as a method in such form. It's just a utility function. It probably could if it used c->GetComponentName() instead of the componentName argument, I'm not sure if it fits all cases.

(but fine for me if others ok)